src/sentry/incidents/metric_alert_detector.py

```diff
@@ -32,7 +32,13 @@ class MetricAlertComparisonConditionValidator(NumericComparisonConditionValidato
 
 
 class MetricAlertConditionGroupValidator(BaseDataConditionGroupValidator):
-    conditions = MetricAlertComparisonConditionValidator(many=True)
+    conditions = serializers.ListField(required=True)
+
+    def validate_conditions(self, value):
+        MetricAlertComparisonConditionValidator(data=value, many=True).is_valid(
```

Comment:
Does it make sense do define this `many=True` behaviour on the base class, and define the validator as a property? Or do you think there will be more customizations to `validate_conditions` in the future that don't fit this model?

As an example of what I mean, in `BaseDataConditionGroupValidator` we could do

```
    def validate_conditions(self, value) -> list:
        self.validator(data=value, many=True).is_valid(raise_exception=True)
```

Then we wouldn't need to define this at all on this subclass. Not sure if the validator needs to explicitly implement something to support `many`?